Form/FormFlow.php

```diff
@@ -233,6 +246,22 @@ public function setInstanceId($instanceId) {
 		$this->instanceId = $instanceId;
 	}
 
+	public function setNewInstance($new) {
+		$this->newInstance = $new;
+	}
+
+	public function isNewInstance() {
+		return $this->newInstance;
+	}
+
+	public function setStale($stale) {
+		$this->stale = $stale;
+	}
+
+	public function isStale() {
+		return $this->stale;
+	}
```

Comment:
The properties `stale` and `newInstance` can be used directly in the code. I don't see a need for these getters and setters.